manifests/params.pp

```diff
@@ -5,12 +5,12 @@
 class selinux::params {
   $module_build_root = "${facts['puppet_vardir']}/puppet-selinux"
 
-  case $facts['osfamily'] {
+  case $facts['os']['family'] {
     'RedHat': {
-      if $facts['operatingsystem'] == 'Amazon' {
+      if $facts['os']['name'] == 'Amazon' {
         $package_name = 'policycoreutils'
       } else {
-        $package_name = $facts['operatingsystemmajrelease'] ? {
+        $package_name = $facts['os']['release']['major'] ? {
           '5'     => 'policycoreutils',
```

Comment:
Do you need to convert this to integers or are they still strings? I think the type should match in the specs.

Reply:
that isn't required. Also facter displays it as string, not integer.